Jenkinsfile

```
@@ -96,7 +96,7 @@ node('docker&&linux') {
         stage('Purge cached CSS') {
             sh '''
             curl -X PURGE https://www.jenkins.io/css/jenkins.css
-            curl -x PURGE https://www.jenkins.io/stylesheets/styles.css
+            curl -X PURGE https://www.jenkins.io/stylesheets/styles.css
```

Post review: long form for flags helps a lot:

```suggestion
curl --request PURGE https://www.jenkins.io/stylesheets/styles.css
```